paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -778,7 +742,7 @@ internal class DefaultFlowController @Inject internal constructor(
         ): PaymentSheet.FlowController {
             val flowControllerViewModel = ViewModelProvider(
                 owner = viewModelStoreOwner,
-                factory = SavedStateViewModelFactory()
+                factory = FlowControllerViewModel.Factory(statusBarColor)
```

Comment:
I think this is a bit of a risky change. And we should do some more testing to ensure it's safe.

Reply:
Some repro steps: https://github.com/stripe/stripe-android/issues/7831